src/main/java/cc/hyperium/utils/HyperiumFontRenderer.java

```diff
@@ -26,13 +26,14 @@
 import java.awt.*;
 import java.util.ArrayList;
 
-public class HyperiumFontRenderer {
+public class HyperiumFontRenderer{
 
     private final UnicodeFont unicodeFont;
     private final int[] colorCodes = new int[32];
     private int fontType;
     private int size;
     private String fontName;
+    public int FONT_HEIGHT = 9;
```

Comment:
Why?

Reply:
This is used in the HyperiumTextField class.